### DISCUSSION TITLE
web scanning

### DISCUSSION THREAD

**Discussion by Teicu** [hktalent/scan4all#135] [Q&A]
Hi all, 

is it possible to start a web scan without subdomain enumeration and port scanning? 

for **scan4all -host https://site.com -ec** I get errors such as: 

_Starting Nmap 7.93 ( https://nmap.org ) at 2023-02-08 16:14 GMT
Read data files from: /usr/local/bin/../share/nmap
Nmap done: 0 IP addresses (0 hosts up) scanned in 0.04 seconds
           Raw packets sent: 0 (0B) | Rcvd: 0 (0B)

WARNING: No targets were specified, so 0 hosts scanned.

[INF] Running SYN scan with root privileges
[INF] Port scan starting....
[INF] Loading the default
[INF] Enumerating subdomains for site.com
[ERR] Could not verbose results for site.com: short write
[FTL] Could not run enumeration: short write_  

I don't know what I'm doing wrong, but nothing works. I use Mac and the latest version of this tool

Thanks

**Comment by hktalent** [comment]
Before starting scanning

1. Check whether the dns is successful. If it fails, the target will not participate in the scan

2. Attempt http request, failed, do not scan
